ArduCopter/Copter.h

```diff
@@ -90,13 +90,15 @@
 #include <AC_InputManager/AC_InputManager.h>        // Pilot input handling library
 #include <AC_InputManager/AC_InputManager_Heli.h>   // Heli specific pilot input handling library
 #include <AP_Button/AP_Button.h>
+#include <AP_Arming/AP_Arming.h>
```

Comment:
Strange, I get compile errors with make when I try to remove this.  The errors appear in defines.h which is odd.  In any case, I'm going to leave this include in place for now.